<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QuickFindFile</class>
 <widget class="QWidget" name="QuickFindFile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search Project File</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Type text to match in filenames (up/down to select shown files)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="fileNameEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>9</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Enter search strings separated by a blank</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="fileList">
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <column>
      <property name="text">
       <string>Path</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Filename</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Open</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>fileNameEdit</tabstop>
  <tabstop>fileList</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QuickFindFile</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>40</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>41</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
